coiote-aws-iot-cloud-formation/certificates/certificates.py

```diff
@@ -0,0 +1,149 @@
+import json
+import os
+from typing import TypedDict
+
+from requests import Response
+
+import boto3
+import requests
+from crhelper import CfnResource
+from OpenSSL import crypto
+
+
+class Certificate(TypedDict):
+    certificatePem: str
+    privateKey: str
+
+
+class InternalCertificate(TypedDict):
```

Comment:
Dodałbym do nazwy jakieś `Payload`/`Body`, żeby zaznaczyć że to definiuje ciało requesta